src/translations/en.json

```diff
@@ -2586,7 +2586,8 @@
             "home_id": "Home ID",
             "close": "Close",
             "add_node": "Add Node",
-            "remove_node": "Remove Node"
+            "remove_node": "Remove Node",
+            "reconfigure_server": "Re-configure Server"
```

Comment:
It is the integration (config entry) you re-configure and not the server?
Is this naming on purpose?

Reply:
It'll change the server options both in the config entry and, if used, in the add-on.